packages/devtools_app/test/test_infra/test_data/dart_tooling_api/mock_api.dart

```diff
@@ -136,6 +144,9 @@ class MockDartToolingApi extends DartToolingApiImpl {
     return true;
   }
 
+  /// Simulates opening a DevTools feature.
+  Future<void> openDevToolsPage(json_rpc_2.Parameters parameters) async {}
```

Comment:
should we at least log something here?

Reply:
We can, although it feels redundant - the request/response will already appear in the log so you can see the request was successful. We could log additional events like "Opened DevTools page foo" but for consistency we'd probably then log the other commands too and it could just result in the log having extra lines for the same things (and it's probably the JSON that is most useful).

Reply:
if the request response are already logged, then that is sufficient 👍